Packs/Microsoft_SharePoint_ToolShell_Vulnerability/Playbooks/playbook-Cortex_XDR_-_CVE-2025-49704_and_CVE-2025-49706_and_CVE-2025-53770_and_CVE-2025-53771_-_Microsoft_SharePoint_ToolShell_vulnerability_chain.yml

```diff
@@ -0,0 +1,3529 @@
+        | alter IIS_appName = arrayindex(regextract(actor_process_command_line, "\-ap\s+\"([^\"]+)\""), 0)
+
+        // Extract fields from the dynamic_string_string_map:
+        // EventSrcIP - Logged IP address by the IIS server
+        // RequestURI - The requested URL by the threat actor
+        // Payload - The time he decoded .NET payload from exploitation
+        // Headers - HTTP request headers
+        | alter EventSrcIP = trim(json_extract(dynamic_event_string_map, "$.27"), "\""),
+                RequestURI = trim(json_extract(dynamic_event_string_map, "$.26"), "\""),
+                Payload = trim(json_extract(dynamic_event_string_map, "$.30"), "\""),
+                Headers = trim(json_extract(dynamic_event_string_map, "$.32"), "\"")
+
+        // Extract the X-Forwarded-For headers from the Headers field in an attempt to identify the source of exploitation
+        | alter x_forwarded_for_header = regextract(lowercase(Headers), "\|(?:client-ip|x-forwarded-for)\:((?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[1-9])(?:\.(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])){3})\|")
+
+        | fields _time, agent_hostname, actor_effective_username, actor_process_image_path, actor_process_command_line, IIS_appName, dynamic_event_string_map, event_thread_context, EventSrcIP, x_forwarded_for_header, RequestURI, Payload, Headers
+        ```
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "97"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3120,
+          "y": 3125
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "133":
+    id: "133"
+    taskid: 22a54ddf-8396-45c5-8e23-dfe79e03ce72
+    type: regular
+    task:
+      id: 22a54ddf-8396-45c5-8e23-dfe79e03ce72
+      version: -1
+      name: Set Potential Agent IDs to search on
+      description: "Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.\n\nThis automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "72"
+    scriptarguments:
```

Comment:
please change the argument 'stringify' to 'True'
<img width="456" height="312" alt="image" src="https://github.com/user-attachments/assets/ef74f114-c97d-4dd6-91ad-f243d787340f" />